src/js/profiling/cache.ts

```diff
@@ -0,0 +1,75 @@
+import type { ThreadCpuProfile } from './types';
+
+/**
+ * Creates a cache that evicts keys in fifo order
+ * @param size {Number}
+ */
+// sentry-javascript/packages/browser/src/profiling/cache.ts
+// https://github.com/getsentry/sentry-javascript/blob/bf8c0551e2a969083199f014d07ac2a617e323fc/packages/browser/src/profiling/cache.ts#L7
```

Comment:
Can we reuse the JS class instead?

Reply:
Yes, that's why I left the TODO comment. [PR here](https://github.com/getsentry/sentry-javascript/pull/8270).